subjects/dress_code/README.md

```diff
@@ -0,0 +1,63 @@
+## dress_code
+
+### Instructions
+
+Create a function called `choose_outfit` that receives the following input:
+- a `formality_level` as an `Option<u32>`;
```

Comment:
- First letter of a line should be Uppercase.
- List items should end with `"."` and not `";"`.

Both of those are just style conventions that I follow, they are not 100% consistent in the piscine, still for new exercises I'm following it quite closely.

Notice this applies to all other list items also.
(special names like `jacket` do not require to have upper case)